Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.py

```diff
@@ -596,39 +599,43 @@ def fetch_credentials():  # pragma: no cover
     engines = argToList(demisto.params().get('engines', []))
     identifier = demisto.args().get('identifier')
     concat_username_to_cred_name = argToBoolean(demisto.params().get('concat_username_to_cred_name') or 'false')
+    concat_folder_to_cred_name = argToBoolean(demisto.params().get('concat_folder_to_cred_name') or 'false')
     if len(engines) == 0:
         return_error('No secrets engines specified')
     for engine_type in engines:
         engines_to_fetch = list(filter(lambda e: e['type'] == engine_type, ENGINE_CONFIGS))
         engines_to_fetch_from += engines_to_fetch
     if len(engines_to_fetch_from) == 0:
         return_error('Engine type not configured, Use the configure-engine command to configure a secrets engine.')
-
+    demisto.debug(f'{engines_to_fetch_from=}')
     for engine in engines_to_fetch_from:
+        folder = engine.get('folder')
+        demisto.debug(f'{folder=}')
         if engine['type'] == 'KV':
             if 'version' not in engine:
                 return_error('Version not configured for KV engine, re-configure the engine')
             if engine['version'] == '1':
-                credentials += get_kv1_secrets(engine['path'], concat_username_to_cred_name)
+                credentials += get_kv1_secrets(engine['path'], concat_username_to_cred_name, folder, concat_folder_to_cred_name)
```

Comment:
Why not passing only the folder name? If a folder given then it is True, otherwise False.